architecture/index.md

```diff
@@ -1,30 +1,44 @@
 # AppStudio
 
 ## Overview
-AppStudio is a platform for building integrated experiences that streamline, consolidate, and secure the application lifecycle.
+AppStudio is a platform for building integrated software that streamlines, consolidates, and secures the development lifecycle.
 
 
 ### Goals
-- Compose cloud native applications that consist of multiple components and services
-- Provide managed application lifecycle
-- Rapid bootstrapping of applications
-- Fast onboarding of applications to the cloud
-- Supports both existing and new applications
-- Provide APIs to manage your application lifecycle
-- Provide a surface for partners to integrate into the application lifecycle
-
-## Architecture Goals and Constraints
-- Robust delivery automation: Establish continuous delivery practices but also deliver operational tooling.
+- Compose software that consists of multiple components, from multiple repositories.
+- Provide transparency on the software supply chain, including both what makes up the software and how it was built.
+- Provide a way for software teams to release to destinations under the control of their SRE or release engineering team(s).
+- Rapid bootstrapping of new software projects
+- Support both existing and new projects
+- Provide APIs to manage your software lifecycle
+- Provide a surface for partners to integrate, add value
+
+## Architecture Goals
+
+- Build artifacts once with enough data to determine suitability for releasing.
+- Build artifacts once that can be released to multiple locations, use cases.
+- "Shift left" the decisions for releasing into PRs; you should be able to release artifacts from a PR as soon as it is merged.
 - Just in time scaling: In contrast to “just in case” scaling. The system should be able to scale without capacity reserved ahead of time.
-- Static stability: the overall system continues to work when a dependency is impaired
-- Each subservice can fulfill its primary use cases independently, without relying on  other systems’ availability.
+- Static stability: the overall system continues to work when a dependency is impaired.
+- Each subservice can fulfill its primary use cases independently, without relying on other systems’ availability.
```

Comment:
is this true in the case of tekton?  doesn't seem like it

if so, should we highlight this exception?

Reply:
Agreed! Lots of things need tekton (that's what you're saying, right?) Yeah, we should call it out.

Reply:
> Agreed! Lots of things need tekton (that's what you're saying, right?) Yeah, we should call it out.

correct that is what I am saying

Reply:
Added in 9332e3b.

Reply:
looks good